tutorial/README.md

```diff
@@ -0,0 +1,168 @@
+```
+
+In order to make this file available to our cluster, we create an archive of the hello folder and upload it to a location from where it available to the cluster. For example, in our example to [https://downloads.mesosphere.com/tensorflow-dcos/examples/hello.zip](https://downloads.mesosphere.com/tensorflow-dcos/examples/hello.zip).
+
+## Prepare the Options File
+
+With the current deployment model for the beta DC/OS TensorFlow package, you have to specify the model you want to run as options during the package installation (as mentioned earlier, this behavior will change in future releases).
+The easiest way of specifying these options is by installing the package using the [DC/OS CLI](https://docs.mesosphere.com/latest/cli/install/).
+
+```bash
+$ dcos package install beta-tensorflow --options=OPTIONS.json
+```
+
+We provide a [script](../scripts/new-config.sh) which helps you generate a template for such option files.
+
+```bash
+$ scripts/new-config.sh helloworld
+```
+
+This command will generate the following template here `examples/local/helloworld..json`
+
+```json
+{
+  "service": {
+    "name": "helloworld",
+    "job_url": "{{JOB_URL}}",
+    "job_path": "",
+    "job_name": "{{JOB_NAME}}",
+    "job_context": "",
+    "shared_filesystem": "",
+    "use_gcs_key_secret": false,
+    "use_tensorboard": false,
+    "user": "root",
+    "tf_image": "mesosphere/dcos-tensorflow:v1.3",
+    "gpu_tf_image": "mesosphere/dcos-tensorflow:v1.3-gpu"
+  },
+  "gpu_worker": {
+    "count": 0,
+    "gpus": 1,
+    "cpus": 1,
+    "mem": 4096,
+    "disk": 4096,
+    "disk_type": "ROOT"
+  },
+  "worker": {
+    "count": 1,
+    "port": 2222,
+    "cpus": 1,
+    "mem": 4096,
+    "disk": 4096,
+    "disk_type": "ROOT"
+  },
+  "parameter_server": {
+    "count": 1,
+    "port": 2223,
+    "cpus": 1,
+    "mem": 4096,
+    "disk": 4096,
+    "disk_type": "ROOT"
+  },
+  "tensorboard": {
+    "port": 6006,
+    "cpus": 1,
+    "mem": 4096
+  }
+}
+```
+All what is left for us to specify are the following variables:
+
+* "job_url": This is the URI from where the previously created archive can be downloaded, in our case: ```"job_url": "https://downloads.mesosphere.com/tensorflow-dcos/examples/hello.zip",```
+* "job_path": This is the folder within the the archive, in our case: ```"job_path": "hello",```
+* "job_name": This is the name for the python file, in our case ```"job_name": "hello_world",```
+
+The final options file can also be found [here](./helloworld.json).
+
+```bash
+$ dcos package install beta-tensorflow --options=./helloworld.json
+```
+Congratulations, you have just trained your first custom TensorFlow Model!
+
+**NOTE:** Once the example has completed, the `ps-*` and `*worker-*` tasks will terminate, but the `EXAMPLE_NAME` scheduler may still remain. This can be removed (or the computation terminated) by running:
+```bash
+$ dcos package uninstall beta-tensorflow --app-id=/EXAMPLE_NAME
```

Comment:
I think it might be more clear here to explicitly write `--app-id=helloworld` for consistency within the example. Similarly above on line 137.